app/Resources/views/emails/shift_late_alerts_default.html.twig

```diff
@@ -0,0 +1,5 @@
+{% for alert in alerts %}
+<strong>Compteur de
+  {% for user in alert.beneficiaries %}{{ user.firstname }} {{ user.lastname }}{% endfor %} :
```

Comment:
je mettrais plutôt `beneficiary` à la place de `user` pour ne pas s'emmêler avec l'entity `User`
idem les `alerts` sont en fait des `Membership`, donc peut-être renommer à `membership_late_alerts` pour clarifier ?

Reply:
Noté je vais changer ça

Reply:
merci pour la capture d'écran. dernière demande : possible de rajouter le `#{{ membership_late_alerts.member_number }}` devant le Prénom Nom ? ou mieux utiliser {{ beneficiary.displayNameWithMemberNumber }} :heart_eyes:

Reply:
j'insiste un peu car on pourrait s'en servir aussi de notre coté de cette commande :)

Reply:
Corrigé, avec `displayNameWithMemberNumber`